pkgs/by-name/ej/ejabberd/package.nix

```diff
@@ -17,10 +17,8 @@
   gd,
   autoreconfHook,
   gawk,
-  rebar3WithPlugins,
   fetchFromGitHub,
   fetchgit,
-  fetchHex,
```

Comment:
fetchHex doesn't use erlang at all, but since it's already in the beamPackages set I moved it for consistency.